includes/Checker/Abstract_Check_Runner.php

```diff
@@ -419,9 +419,7 @@ final public function get_checks_to_run() {
 			$collection = Check_Categories::filter_checks_by_categories( $collection, $categories );
 		}
 
-		return $collection
-			->require( $check_slugs ) // Ensures all of the given slugs are valid.
```

Comment:
You have removed `require( $check_slugs )`. Are we sure we want to remove validation altogether?

Reply:
Missed that completely nice catch. Reverted in https://github.com/WordPress/plugin-check/pull/387/commits/9fb14284cea537776679592dfd7435965e149b97